docs/_includes/js/modal.html

```diff
@@ -218,6 +218,29 @@ <h2 id="modals-remove-animation">Remove animation</h2>
 </div>
 {% endhighlight %}
 
+  <h2 id="modals-grid-system">Using the grid system</h2>
+  <p>For modals which would like to take advantage of the Bootstrap grid system, nest <code>.container-fluid</code> within the <code>.modal-body</code> to have access to all the grid system classes</p>
+{% highlight html %}
+<div class="modal fade">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        ...
+      </div>
+      <div class="modal-body">
+        <div class="container-fluid">
+          ...
```

Comment:
Show a couple rows with a couple columns in them so the example is more complete and unambiguous.